app/routes/products.$productSlug/route.tsx

```diff
@@ -129,6 +128,7 @@ export default function ProductDetailsPage() {
                                 option={option}
                                 selectedChoice={selectedChoices[option.name!]}
                                 onChange={(newSelectedChoice) => {
+                                    setQuantity(1);
```

Comment:
why do we want to reset quantity on option change?
also, are you going to handle quantity in stock in future PRs?

Reply:
>why do we want to reset quantity on option change?

this is how wix template behaves.. kind of make sense I guess

> also, are you going to handle quantity in stock in future PRs?

not for now